drivers/hmis/app/graphql/types/hmis_schema/has_enrollments.rb

```diff
@@ -23,17 +23,17 @@ def enrollments_field(name = :enrollments, description = nil, type: [Types::Hmis
       end
 
       def resolve_enrollments_with_loader(association_name = :enrollments, **args)
-        load_ar_association(object, association_name, scope: apply_enrollment_arguments(Hmis::Hud::Enrollment, **args))
+        load_ar_association(object, association_name, scope: scoped_enrollments(Hmis::Hud::Enrollment, **args))
       end
 
       def resolve_enrollments(scope = object.enrollments, **args)
-        apply_enrollment_arguments(scope, **args)
+        scoped_enrollments(scope, **args)
       end
 
       private
 
-      def apply_enrollment_arguments(scope, user: current_user, sort_order: :most_recent)
-        scope = scope.viewable_by(user)
+      def scoped_enrollments(scope, sort_order: :most_recent)
+        scope = scope.viewable_by(current_user)
```

Comment:
for consistency, we should choose if we want to apply the `viewable_by` first or last in these methods, in assessments it is first, in client and enrollment, last, seems like it'd be nice to pick one.